### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by andyw8**
Use latest haml, haml_lint and rubocop.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -9,6 +9,22 @@ AllCops:
     - "db/schema.rb"
   TargetRubyVersion: 2.1
 
+Layout/DotPosition:
```
Comment (andyw8) [inline .rubocop.yml:12]: These changed namespace in the latest RuboCop.

File: lib/tasks/docs.rake [hunk 2]
```diff
@@ -58,19 +58,16 @@ module ExtractDocumentation
           block[:body] << line
           in_documentation_block = true
         end
+      elsif in_documentation_block
```
Comment (andyw8) [inline lib/tasks/docs.rake:61]: As suggested by RuboCop.

**Review by michaelherold — APPROVED** [review]
This looks great, thanks!

I'm going to merge these commits with a few extra changes (mostly to engine.json). I hope I'll get it done before a meetup in 20 minutes, otherwise, I'll follow up tomorrow.

**Comment by michaelherold** [comment]
Merged and released! I'll have to pop into the CodeClimate Slack tomorrow to see what I have to do to get it updated in their system.

**Pull request merged**